json-definitions/v3/post/complete/request/index.json

```diff
@@ -0,0 +1,218 @@
+{
+  "title": "Tech Record POST Request Complete Car Schema",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "msUserDetails",
+    "vin",
+    "primaryVrm",
+    "trailerId",
+    "techRecord_reasonForCreation",
+    "techRecord_vehicleType",
+    "techRecord_statusCode",
+    "techRecord_regnDate",
+    "techRecord_manufactureYear",
+    "techRecord_noOfAxles",
+    "techRecord_vehicleSubclass",
+    "techRecord_vehicleConfiguration",
+    "techRecord_euVehicleCategory",
+    "techRecord_applicantDetails",
+    "techRecord_createdAt",
+    "techRecord_createdByName",
+    "techRecord_createdById",
+    "techRecord_lastUpdatedAt",
+    "techRecord_lastUpdatedByName",
+    "techRecord_lastUpdatedById"
+  ],
+  "properties": {
+    "msUserDetails": {
```

Comment:
Didn't discuss it with you, however we would like (and I think it's possible through some extra config in APIGW) to grab the msUserDetails from the JWT the user sends with the request. Mostly because it's "the proper" way of doing it. Also because in theory some authorised user could amend records and use someone else's name in there, which from an auditing point of view is not ideal, not that this is likely to happen in practice. This would also lessen the burden on clients to supply that information.

Reply:
Does that mean that it will need to be removed from the schema if it's being done in the APIGW?

Reply:
It doesn't need to be passed in so i would say no